scripts/database-exporter/publish.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Generates and publishes a database export for use with OOB catchup:
+# Once the export has been generated and verified,
+# it will be uploaded to the specified AWS S3 bucket with public access
+# and the relevant entry of the specified AWS CloudFront distribution will be invalidated.
+
+# Expected environment variables:
+# - DB_PATH: Local path to the node's LMDB database (default: '/var/lib/concordium/data/database-v4')
+# - S3_BUCKET_NAME: Name of the S3 bucket (without 's3://' prefix) to which the export should be published.
+# - CF_DISTRIBUTION_ID: ID of the CloudFront distribution.
+
+db_path="${DB_PATH-/var/lib/concordium/data/database-v4}"
```

Comment:
Is this missing a `:`, i.e., `${DB_PATH:-/...}`?

Reply:
Both syntaxes are valid. If `:` is included it will also pick the default if the variable is declared with empty value. I tend to not include it.